hosts/tunnel.go

```diff
@@ -65,6 +65,9 @@ func checkDockerVersion(ctx context.Context, h *Host, clusterVersion string) err
 		return fmt.Errorf("Can't retrieve Docker Info: %v", err)
 	}
 	logrus.Debugf("Docker Info found: %#v", info)
+	if h.IgnoreDockerVersion {
```

Comment:
docker info is not populated if this block returns here. Needs to be moved below.